README.rst

```diff
@@ -47,51 +45,35 @@ Get Involved
    kinds of ways to contribute to and interact with the project,
    including mailing list information and how to submit bug reports and
    code to Ansible.
-*  All code submissions are done through pull requests to the ``devel`` branch.
-*  Feel free to talk to us before making larger changes
+*  Join a `Working Group
+   <https://github.com/ansible/community/wiki>`_, an organized community devoted to a specific technology domain or platform.
+*  Submit proposed a code update through a pull request to the ``devel`` branch.
```

Comment:
probably worth linking to https://help.github.com/en/articles/about-pull-requests